1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -18,6 +18,12 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+
+    let i = 0;
+    while (i < BIRTHDAYS.length) {
+      console.log(i[5]);
```

Comment:
That also was my first  thought too, however there is method .includes which will work well in this exercise and will save some time , if you have an array with hundreds of strings.